onset_maproom/tests/test_calc.py

```diff
@@ -4,10 +4,463 @@
 
 import calc
 
-def test_onset():
+
+def set_multi_year_data_sample():
```

Comment:
This doesn't set anything. Call it `get_multi_year_data_sample` or just `multi_year_data_sample`.